### PR TITLE
Fix up command window instructions

### PR DESCRIPTION
**Pull request by jhodgdon-drp**
Fix for #1324: Centralize the command window instructions in one include file, including all the best instructions from elsewhere in the tutorial, and also some new instructions for how to get a second command window open in Windows. Include this new include file in both the intro to command line page and the one-page Installation guide. Therefore, remove info on the command line from the Python install instructions include file. Also add a link in the start Django project page where it asks you to open a second command window.

### CODE REVIEW

**Comment by jhodgdon-drp** [comment]
OK, I think I have resolved the conflict.

**Comment by jhodgdon-drp** [comment]
I've been away for a while... coming back to this PR. I just reviewed it again, and I think it would really improve how we give instructions to Windows users about the command line. Basically, it creates a plug-in section for how to open the command line, which collects all the Windows, Mac, and Linux instructions (the various possible Windows ones were scattered around) in one place. Then this is inserted into the one-page install everything guide, as well as the separate command-line topic.

**Comment by jhodgdon-drp** [comment]
Also, this PR says "1 review requesting changes", but the only changes I can see that were requested were to resolve the conflicts, and that has been done. Thanks!

File: en/intro_to_command_line/open_instructions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!--sec data-title="Opening: Windows" data-id="windows_prompt" data-collapse=true ces-->
+
+Depending on your version of Windows and your keyboard, one or more of the following should work to open a command window:
```
[the remark this answers is not in the document]
Comment (jhodgdon-drp) [reply en/intro_to_command_line/open_instructions.md]: Good thought. I've edited the text, and reordered the suggestions. How's this?

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**